docs/README.md

```diff
@@ -82,93 +82,79 @@ following:
 
 </details>
 
-## Current Features
-
-- Discord Gateway communication
-- logging
-- Http Client
-- Events
-- Event middleware
-- Commands
-- Command arguments *(for types: str, int, float, bool, User, Channel, Role)*
-- Command argument choices
-- Command argument descriptions
-- Command cool downs (Using WindowSliding technique)
-- Tasks
-- Cogs
-
 **Client base class example:**
 
 ```py
 from pincer.client import Bot
 
 # Note that both `Bot` and `Client` are valid!
-bot = Bot("...")
+bot = Bot("YOUR_TOKEN_HERE")
 bot.run()
 ```
 
 **An example on the `on_ready` event**
 
+Pincer makes developing application commands intuitive and fast.
+
 ```py
 from time import perf_counter
-from pincer.client import Client
-
-client = Client("...")
-
-
-@client.event
-async def on_ready():
-    print(f"Logged in as {client.bot} after {perf_counter()} seconds")
+from pincer import Client
 
+class Bot(Client):
+    @Client.event
+    async def on_ready():
+        print(f"Logged in as {client.bot} after {perf_counter()} seconds")
```

Comment:
also `perf_counter()` is broken on my server idk, it put like 3 milion seconds, work fine when subtracting from a marker

Reply:
It works fine locally? Idk what Im supposed to do lol

Reply:
nvm its supposed to be ms not seconds